patches/elasticpress.0003.fix.php-warning-for-cli-batch-time-length-calculation.patch

```diff
@@ -1,26 +1,26 @@
-From efe35a4b7b545788876f3ec3d8281d5f7c47826e Mon Sep 17 00:00:00 2001
-From: Nabi <artsanaat@gmail.com>
-Date: Thu, 29 Feb 2024 14:21:18 +0330
-Subject: [PATCH] Fixed PHP warning for batch time length calculation in
- ElasticPress CLI.
+From b377a1313cb023c022167921038dbab8cc8aaee4 Mon Sep 17 00:00:00 2001
+From: navidabdi <artsanaat@gmail.com>
+Date: Fri, 21 Mar 2025 12:47:11 +0400
+Subject: [PATCH] fix: PHP warning for batch time length calculation in
+ ElasticPress CLI
 
 ---
  includes/classes/Command.php | 2 +-
  1 file changed, 1 insertion(+), 1 deletion(-)
 
 diff --git a/includes/classes/Command.php b/includes/classes/Command.php
-index 81fa2f2c5..a1c6c2e93 100644
+index 0ae7e9bcd..55b30596d 100644
 --- a/includes/classes/Command.php
 +++ b/includes/classes/Command.php
-@@ -1335,7 +1335,7 @@ class Command extends WP_CLI_Command {
+@@ -1331,7 +1331,7 @@ class Command extends WP_CLI_Command {
  		if ( 'index_next_batch' === $context ) {
- 			$counter++;
+ 			++$counter;
```

Comment:
This seem to be fixed in the current version
https://github.com/10up/ElasticPress/blob/5.1.4/includes/classes/Command.php#L1332

Reply:
Note that there is no functional difference in this change.  There would only be a difference if the increment was part of the condition in the following line.